### PR TITLE
fix: eslint error

### PR DESCRIPTION
**Pull request by LiKang6688**
fix a lint issue in https://app.circleci.com/pipelines/github/qlik-oss/sn-network-chart/150/workflows/5cc68fd0-e1a4-47a3-a80b-90eb19941613/jobs/324

### CODE REVIEW

**Review by Ashishyadav13 — APPROVED** [review]

**Review by Ashishyadav13 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
     "build": "nebula build --core core",
     "sense": "nebula sense --meta resources/meta.json && shx cp resources/network_chart_v1.png sn-network-chart-ext",
     "eslint": "eslint src",
+    "test:unit": "jest",
```
Comment (niekvanstaveren) [inline package.json:16]: For some reason unit tests are not included in the pipeline. Could you add them as well while you are working on this?

Comment (LiKang6688) [reply package.json:16]: Good point. Let me add it.

Comment (LiKang6688) [reply package.json]: Done

**Review by niekvanstaveren — APPROVED** [review]
Not completely sure about the order for the pipeline, but at least things are running. Great job!

**Pull request merged**